runtime/core/src/main/java/io/atlasmap/core/DefaultAtlasConversionService.java

```diff
@@ -510,9 +506,7 @@ public FieldType fieldTypeFromClass(String className) {
         case DATE:
             return java.time.LocalDate.class;
         case DATE_TIME:
-            return java.util.Date.class;
-            // TODO do we prefer java.time.* at some point? - https://github.com/atlasmap/atlasmap/issues/312
-            // return java.time.LocalDateTime.class;
+            return java.time.LocalDateTime.class;
```

Comment:
Is this one required for any reason? I think java.util.Date is still a majority so I'd vote to stick with java.util.Date as a first representative for now.